docs/data-cookbooks/setup.md

```diff
@@ -47,6 +47,7 @@ The schema for collections can be found [here](https://github.com/nasa/cumulus/t
 |:---:|:-----:|:--------:|-----------|
 |regex|`"^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.hdf$"`|Yes|Regex used to identify the file|
 |sampleFileName|`MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"`|Yes|Filename used to validate the provided regex|
+|fileType|`"data"`|No|Value to be assigned to the Granule File Filetype.  CNM filetypes used by Cumulus CMR steps, non-CNM values will be treated as 'data' fileType|
```

Comment:
A couple thoughts:

- Doesn't seem like the words `Granule File Filetype` should be capitalized?
- `CNM filetypes used by Cumulus CMR steps` should be `CNM filetypes will be used by Cumulus as fileType for CMR steps`?

Reply:
```suggestion
|fileType|`"data"`|No|Value to be assigned to the granule file fileType.  CNM file types will be used by Cumulus CMR steps as the fileType, non-CNM values will be treated as the 'data' fileType|
```